modules/base/rendering/renderableplanet.cpp

```diff
@@ -349,7 +349,7 @@ void RenderablePlanet::render(const RenderData& data) {
     glm::dmat4 rot = glm::rotate(glm::dmat4(1.0), M_PI_2, glm::dvec3(1, 0, 0));
     glm::dmat4 roty = glm::rotate(glm::dmat4(1.0), M_PI_2, glm::dvec3(0, -1, 0));
     //glm::dmat4 rotProp = glm::rotate(glm::dmat4(1.0), glm::radians(static_cast<double>(_rotation)), glm::dvec3(0, 1, 0));
-    modelTransform = modelTransform * rot /** roty*/ /** rotProp*/;
+    modelTransform = modelTransform * rot * roty /** rotProp*/;
```

Comment:
There is a comment right before these lines saying "earth needs to be rotated for that to work." Can these rotations be explained more clearly? Why is there a rotation of pi around x and then around y? That should be the same as one single rotation around z, am I right?